R/github-actions.R

```diff
@@ -227,12 +248,17 @@ use_tidy_github_actions <- function(ref = NULL) {
 
   use_github_action("check-full.yaml", ref = ref, badge = TRUE)
 
-  use_github_action("pr-commands", ref = ref)
   use_github_action("pkgdown", ref = ref)
 
   use_coverage(repo_spec = repo_spec)
   use_github_action("test-coverage", ref = ref)
 
+  use_github_action(
+    url = "https://github.com/posit-dev/setup-air/blob/main/examples/format-suggest.yaml"
+  )
```

Comment:
Random thought that possibly it should be guarded with `if (uses_air()) {`? Which would look for an `air.toml` at root level?

I'm not sure, I think what is here is also fine as it encourages you to opt in to air with `use_air()` as well.

Reply:
Yeah I'll do this.

Reply:
In a setting where such hyperlinks work, `use_air()` is clickable/runnable here:

> ! Can't find an air.toml file. Do you need to run use_air()?